inc/saiport.h

```diff
@@ -438,7 +438,7 @@ typedef enum _sai_port_attr_t
      * @brief Media Type
      *
      * @type sai_port_media_type_t
-     * @flags CREATE_ONLY
+     * @flags CREATE_AND_SET
```

Comment:
no it does not require to recreate the port again. what other attributes? HW_LANE_LIST is related to break out so it is create only. even you replace the port module, it port will not automatically break out into 4, you still need to call remove_port() and create_port()